packages/client/src/containers/MovieListPage/ReducerForFetchMoviesAndCategories.jsx

```diff
@@ -0,0 +1,101 @@
+import { apiURL } from '../../apiURL';
+
+export const initialState = {
+  movieLoading: false,
+  categoryLoading: false,
+  moviesData: {},
+  categories: [],
+  movieFetchError: '',
+  categoryFetchError: '',
+  searchText: '',
+  categoryId: null,
+  sortBy: 'rating',
+};
+
+export const handleMovieFetch = async (state, dispatchFunc) => {
+  let URI = `${apiURL()}/movies?`;
```

Comment:
This is very nicely done and structured, very clean and clear. 
It makes it very easy for anybody to see all the possible movie fetches that can happen and makes it easy to work with this 👏🏽